src/components/molecules/EventPaymentButton/EventPaymentButton.tsx

```diff
@@ -1,89 +0,0 @@
-import React from "react";
-import "firebase/functions";
-import { Link } from "react-router-dom";
-
-import { VenueEvent } from "types/venues";
-
-import { hasUserBoughtTicketForEvent } from "utils/hasUserBoughtTicket";
-import { isUserAMember } from "utils/isUserAMember";
-import { isEventStartingSoon } from "utils/event";
-import { WithId } from "utils/id";
-import { venueEntranceUrl } from "utils/url";
-import {
-  currentVenueSelectorData,
-  userPurchaseHistorySelector,
-} from "utils/selectors";
-
-import { useUser } from "hooks/useUser";
-import { useSelector } from "hooks/useSelector";
-import useConnectUserPurchaseHistory from "hooks/useConnectUserPurchaseHistory";
```

Comment:
I don't see any changes to `hooks/useConnectUserPurchaseHistory ` in this PR. I'm guessing this can probably be removed now? Along with any of the related types in `src/types/Firestore.ts`, and then any selectors/usages/etc that relied on any of this.

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40